replica/database.cc

```diff
@@ -2925,7 +2925,7 @@ mutation_reader make_multishard_streaming_reader(distributed<replica::database>&
         auto table_id = s->id();
         const auto buffer_hint = multishard_reader_buffer_hint(multishard_reader_buffer_size.has_value());
         auto rd = make_multishard_combining_reader_v2(seastar::make_shared<streaming_reader_lifecycle_policy>(db, table_id, compaction_time),
-                std::move(s), erm, std::move(permit), pr, ps, std::move(trace_state), fwd_mr, buffer_hint);
+                std::move(s), erm, std::move(permit), pr, ps, std::move(trace_state), fwd_mr, buffer_hint, read_ahead::no);
```

Comment:
I think this should also be a parameter, in case it turns out to be a bad choice for some environments.